backend/composer/models.py

```diff
@@ -229,17 +229,19 @@ class AnatomicalEntity(models.Model):
     """Anatomical Entity"""
 
     name = models.CharField(max_length=200, db_index=True)
-    ontology_uri = models.URLField()
+    ontology_uri = models.URLField(unique=True)
 
     def __str__(self):
         return self.name
 
     class Meta:
         ordering = ["name"]
         verbose_name_plural = "Anatomical Entities"
-        constraints = [
-            models.UniqueConstraint(Upper("name"), name="ae_unique_upper_name")
-        ]
+
+
+class Synonym(models.Model):
+    anatomical_entity = models.ForeignKey(AnatomicalEntity, on_delete=models.CASCADE, related_name="synonyms")
+    alias = models.CharField(max_length=200, db_index=True)
```

Comment:
suggestion: @afonsobspinto rename alias to name to be more in line of other models. The model name Synonym already explains what the data is

Reply:
critical: @afonsobspinto imo the "name" of the synonym should be unique. At least within the AE.
question: @afonsobspinto what if the name of the synonym also exists in another AE?

Reply:
Agree with the renaming and with the unique together. ✅ 

For the question, the data suggests that having the same name/synonym for different uris is something that can happen (see example below). 
I would say the onus of dealing with it properly is on the curator but we can ask the client cc @ddelpiano .
```
http://purl.obolibrary.org/obo/UBERON_0000019,camera-type eye,eye
http://purl.obolibrary.org/obo/UBERON_0010230,eyeball of camera-type eye,eye
```